exercises/go-counting/README.md

```diff
@@ -0,0 +1,50 @@
+# Go Counting
+
+Count the scored points on a Go board.
+
+In the game of go (also known as baduk, igo, cờ vây and wéiqí) points
+are gained by completely encircling empty intersections with your
+stones. The encircled intersections of a player are known as its
+territory.
+
+Write a function that determines the territory of each player. You may
+assume that any stones that have been stranded in enemy territory have
+already been taken off the board.
+
+Write a function that determines the territory which includes a specified coordinate.
```

Comment:
Did you generate this readme using configlet? It doesn't appear to be consistent with the one in [problem specifications](https://github.com/exercism/problem-specifications/blob/master/exercises/go-counting/description.md). Specifically this line is not included.

We want our exercises to be consistent with the description and canonical data (if present) in problem specifications. That way the exercises are consistent across tracks 🙂

Reply:
Previously I had submitted a PR to the problem specifications repository which includes an update to the readme. I think that is why it is a little different. I think soon my PR will be merged, but it hasn't been yet. You can take a look at my PR though if you like.

Reply:
Okay, that's great that you've suggested improvements to the problem specifications repo 🙂

However since changes to that repo affect all tracks, PRs can take a while to merge. So I think this reame should be consistent with what problem specifications currently has. 

Feel free to open a new PR to change it on this track once your PR in problem specifications has been merged though 🙂 And we do try to regularly sync our reames with the ones in problem specifications so it should be brought up to date eventually anyway 🙂

Reply:
Though I see now that [your PR to problem specifications](https://github.com/exercism/problem-specifications/pull/1195) also includes canonical data which is great! And it looks like it's close to being merged 🙂 So maybe it would be best to wait until that PR has been merged and then we can review this exericse while having the canonical data and the updated description in problem specifications? 🙂